rapids-cmake/cpm/detail/generate_patch_command.cmake

```diff
@@ -79,6 +87,7 @@ function(rapids_cpm_generate_patch_command package_name version patch_command)
       rapids_cpm_json_get_value(${patch_data} file)
       rapids_cpm_json_get_value(${patch_data} issue)
 
+      cmake_language(EVAL CODE "set(file ${file})")
```

Comment:
I assume that this needs to be an EVAL and not a direct set because it's inside a nested function?

Reply:
JSON variables are read as string literals by CMake so any placeholders won't be evaluated. We use `EVAL` so that any placeholders inside `file` get expanded.